conda/cli/main_list.py

```diff
@@ -113,7 +114,7 @@ def print_export_header():
 
 def get_packages(installed, regex):
     pat = re.compile(regex, re.I) if regex else None
-    for dist in sorted(installed, key=lambda x: x.quad[0].lower()):
+    for dist in sorted(installed, key=lambda x: Dist(x).quad[0].lower()):
```

Comment:
This concerns me: we're jumping through all these hoops to get rid of Dist, but this `quad` monstrosity (of my creation) is still here.